<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpaceToolsConstellationMaker</class>
 <widget class="QWidget" name="SpaceToolsConstellationMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constellation Maker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mOverallLayout">
     <item>
      <layout class="QGridLayout" name="mParametersLayout" columnstretch="1,1,1,1">
       <item row="0" column="0">
        <widget class="QLabel" name="mConstellationNameLabel">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Constellation Name:</string>
         </property>
         <property name="buddy">
          <cstring>mConstellationLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mConstellationLineEdit">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maxLength">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="mPlatFormTypeLabel">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Platform Type:</string>
         </property>
         <property name="buddy">
          <cstring>mPlatformTypeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="mPlatformTypeLineEdit">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="mInclinationLabel">
         <property name="text">
          <string>Inclination:</string>
         </property>
         <property name="buddy">
          <cstring>mInclinationLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="UtQtAngleValueEdit" name="mInclinationLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="mNumberOfPlanesLabel">
         <property name="text">
          <string>Number of Planes:</string>
         </property>
         <property name="buddy">
          <cstring>mNumberOfPlanesLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mNumberOfPlanesLineEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="mSatellitesPerPlaneLabel">
         <property name="text">
          <string>Satellites Per Plane:</string>
         </property>
         <property name="buddy">
          <cstring>mSatellitesPerPlaneLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="mSatellitesPerPlaneLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="mInitialRAAN_Label">
         <property name="text">
          <string>Initial RAAN:</string>
         </property>
         <property name="buddy">
          <cstring>mInitialRAAN_LineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="UtQtAngleValueEdit" name="mInitialRAAN_LineEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="mRAAN_RangeLabel">
         <property name="text">
          <string>RAAN Range:</string>
         </property>
         <property name="buddy">
          <cstring>mRAAN_RangeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="UtQtAngleValueEdit" name="mRAAN_RangeLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="mInitialAnomalyLabel">
         <property name="text">
          <string>Initial Anomaly:</string>
         </property>
         <property name="buddy">
          <cstring>mInitialAnomalyLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="UtQtAngleValueEdit" name="mInitialAnomalyLineEdit"/>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="mAnomalyAliasLabel">
         <property name="text">
          <string>Anomaly Alias:</string>
         </property>
         <property name="buddy">
          <cstring>mAnomalyAliasLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="UtQtAngleValueEdit" name="mAnomalyAliasLineEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="SpaceTools::OrbitSizeWidget" name="mOrbitSize" native="true">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="mWarningLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mBasicControlsLayout">
       <item>
        <layout class="QVBoxLayout" name="mConstellationControlsLayout">
         <item>
          <widget class="QLabel" name="mConstellationOutputLabel">
           <property name="text">
            <string>./scenarios/</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="mConstellationLayout">
           <item>
            <widget class="QPushButton" name="mConstellationPushButton">
             <property name="text">
              <string>Constellation</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="mConstellationSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="mOtherControlsLayout">
         <item>
          <widget class="QPushButton" name="mCancelPushButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAdvancedPushButton">
           <property name="text">
            <string>Advanced</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="mAdvancedWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="mGeneratorOutputLabel">
          <property name="text">
           <string>./scripts/</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mGeneratorLayout">
          <item>
           <widget class="QPushButton" name="mGeneratorPushButton">
            <property name="text">
             <string>Generator</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="mGeneratorSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mPathLayout">
          <item>
           <widget class="QLabel" name="mPathLabel">
            <property name="text">
             <string>File Generation Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mPathLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtAngleValueEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtUnitValueEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>SpaceTools::OrbitSizeWidget</class>
   <extends>QWidget</extends>
   <header>OrbitSizeWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mConstellationLineEdit</tabstop>
  <tabstop>mPlatformTypeLineEdit</tabstop>
  <tabstop>mOrbitSize</tabstop>
  <tabstop>mInclinationLineEdit</tabstop>
  <tabstop>mNumberOfPlanesLineEdit</tabstop>
  <tabstop>mSatellitesPerPlaneLineEdit</tabstop>
  <tabstop>mInitialRAAN_LineEdit</tabstop>
  <tabstop>mRAAN_RangeLineEdit</tabstop>
  <tabstop>mInitialAnomalyLineEdit</tabstop>
  <tabstop>mAnomalyAliasLineEdit</tabstop>
  <tabstop>mConstellationPushButton</tabstop>
  <tabstop>mCancelPushButton</tabstop>
  <tabstop>mAdvancedPushButton</tabstop>
  <tabstop>mGeneratorPushButton</tabstop>
  <tabstop>mPathLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
